src/sentry/sentry_metrics/querying/visitors.py

```diff
@@ -252,3 +368,45 @@ def _visit_timeseries(self, timeseries: Timeseries) -> set[str]:
             raise InvalidMetricsQueryError("Can't determine queried metrics without a MRI")
 
         return {timeseries.metric.mri}
+
+    def _visit_number(self, number: float) -> set[str]:
+        return set()
+
+    def _visit_string(self, string: str) -> set[str]:
+        return set()
+
+
+class UsedGroupBysVisitor(QueryExpressionVisitor[set[str]]):
```

Comment:
nit: i dont understand the naming of this class, should it be just `GroupBysVisitor`?

Reply:
It's because it's not visiting group bys, it's returning you the group bys that were used.

Reply:
So the result groups?

Reply:
Basically

Reply:
got it, thanks! ok then `ResultGroupsVisitor` could be a better name :D